### DISCUSSION TITLE
Consumer assignment for '$G >COMSUMER_NAME > f2YFg5Ot' not cleaned up

### DISCUSSION THREAD

**Discussion by VuongUranus** [nats-io/nats-server#4509] [Q&A]
nats-server after a while will increase disk space and often encounter the warning "Consumer assignment for '$G >COMSUMER_NAME > f2YFg5Ot' not cleaned up, retrying.". At this time, I often get the error "context deadline exceeded" when subscribing to ephemeral consumers. Please help me.

**Comment by derekcollison** [comment]
What server version?

That can be normal if the meta layer is under alot of pressure, since this is another server who is a consumer leader cleaning up an inactive consumer. It asks the meta-layer to get rid of it and watches to make sure that happens.

**Reply by VuongUranus** [reply]
I am using nats-server version v2.9.21. Is there any way to solve this problem?

**Reply by derekcollison** [reply]
What does `nats server report jetsream` show from the system account?

If the meta layer has a leader these will either not happen or clear quickly as the meta-layer reacts.
